src/ConnectedKubernetes/test/Remove-AzConnectedKubernetes.Tests.ps1

```diff
@@ -0,0 +1,31 @@
+$loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
```

Comment:
`New-AzConnectedKubernetes` and `Remove-AzConnectedKubernetes` should be in `live` mode and it's not nessary to checkin the record file.

Reply:
Add Skip tag for test cases for New-AzConnectedKubernetes and Remove-AzConnectedKubernetes